modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -0,0 +1,827 @@
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . 'star_rs_layer_' . $self->o('release_number'),
+      -host   => $self->o('star_rnaseq_for_layer_db_server'),
+      -port   => $self->o('star_rnaseq_for_layer_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'star_rnaseq_for_layer_nr_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_star_rs_layer_nr_' . $self->o('release_number'),
+      -host   => $self->o('star_rnaseq_for_layer_db_server'),
+      -port   => $self->o('star_rnaseq_for_layer_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'scallop_initial_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_scallop_initial_' . $self->o('release_number'),
+      -host   => $self->o('scallop_initial_db_server'),
+      -port   => $self->o('scallop_initial_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'scallop_blast_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_scallop_blast_' . $self->o('release_number'),
+      -host   => $self->o('scallop_blast_db_server'),
+      -port   => $self->o('scallop_blast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+  };
+}
+
+sub pipeline_create_commands {
+  my ($self) = @_;
+
+  my $tables;
+  my %small_columns = (
+    paired      => 1,
+    read_length => 1,
+    is_13plus   => 1,
+    is_mate_1   => 1,
+  );
+  # We need to store the values of the csv file to easily process it. It will be used at different stages
+  foreach my $key ( @{ $self->default_options->{'file_columns'} } ) {
+    if ( exists $small_columns{$key} ) {
+      $tables .= $key . ' SMALLINT UNSIGNED NOT NULL,'
+    }
+    elsif ( $key eq 'DS' ) {
+      $tables .= $key . ' VARCHAR(255) NOT NULL,'
+    }
+    else {
+      $tables .= $key . ' VARCHAR(50) NOT NULL,'
+    }
+  }
+  $tables .= ' KEY(SM), KEY(ID)';
+
+  return [
+    # inheriting database and hive tables' creation
+    @{ $self->SUPER::pipeline_create_commands },
+
+    $self->db_cmd( 'CREATE TABLE ' . $self->o('summary_csv_table') . " ($tables)" ),
+
+    $self->db_cmd( 'CREATE TABLE ' . $self->o('read_length_table') . ' (' .
+        'fastq varchar(50) NOT NULL,' .
+        'read_length int(50) NOT NULL,' .
+        'PRIMARY KEY (fastq))' ),
+
+    'mkdir -p ' . $self->o('rnaseq_dir'),
+    'mkdir -p ' . $self->o('genome_dumps'),
+  ];
+}
+
+sub pipeline_wide_parameters {
+  my ($self) = @_;
+
+  return {
+    %{ $self->SUPER::pipeline_wide_parameters },
+    skip_rnaseq          => $self->o('skip_rnaseq'),
+    wide_ensembl_release => $self->o('ensembl_release'),
```

Comment:
This can be removed as it is not used anymore

Reply:
Removed.